### DISCUSSION TITLE
Patch a font without changing anything else

### DISCUSSION THREAD

**Discussion by ch-47** [ryanoasis/nerd-fonts#1339] [Q&A] [answered]
I'd just like to add extra glyphs to a font, is it possible to patch it with the patcher script, and keep everything else (file name, TTF names, comments) as is?

**Comment by Finii** [comment, marked as answer]
On camping ... so very short:

Try `--makegroups -1`


________________________________
From: ch-47 ***@***.***>
Sent: Wednesday, 16 August 2023 05:02
To: ryanoasis/nerd-fonts
Cc: Subscribed
Subject: [ryanoasis/nerd-fonts] Patch a font without changing anything else (Discussion #1339)

I'd just like to add extra glyphs to a font, is it possible to patch it with the patcher script, and keep everything else (file name, TTF names, comments) as is?

**Reply by ch-47** [reply]
Yeah, that's it, thank you. The font file itself is still renamed though, but it's not a big deal.
